Introduction.Rmd

```diff
@@ -137,64 +159,97 @@ Make sure you have a recent version of RStudio. You can check you have the right
 
 ```{r, eval = FALSE}
 install.packages("rstudioapi")
+# TODO: update this version number?
```

Comment:
What is the logical basis for setting this minimum?

Reply:
I think there was something we needed in the distant past. I think you can now remove.